Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -1953,11 +1953,32 @@ namespace Units
 open MulOpposite (continuous_op continuous_unop)
 
 variable [Monoid α] [TopologicalSpace α] [Monoid β] [TopologicalSpace β]
-
 @[to_additive]
-instance [ContinuousMul α] : TopologicalGroup αˣ where
+instance : ContinuousInv αˣ where
   continuous_inv := Units.continuous_iff.2 <| ⟨continuous_coe_inv, continuous_val⟩
 
+/-- The inversion map on `Units` is a homeomorphism. -/
+@[to_additive (attr := simps!) "The negation map on `AddUnits` is a homeomorphism."]
+protected def homeomorph_inv : αˣ ≃ₜ αˣ where
```

Comment:
As I said before, we already have [Homeomorph.inv](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Algebra/Group/Basic.html#Homeomorph.inv) that works for any type with involutive continuous `Inv.inv`, no need to define a new copy of it.